src/OrbitClientModel/CaptureDeserializer.cpp

```diff
@@ -160,10 +160,22 @@ ErrorMessageOr<CaptureListener::CaptureOutcome> LoadCaptureInfo(
     orbit_grpc_protos::InstrumentedFunction instrumented_function;
     instrumented_function.set_function_id(function.first);
     instrumented_function.set_function_name(function.second.pretty_name());
-    instrumented_function.set_file_path(function.second.loaded_module_path());
-    const ModuleData* module_data =
-        module_manager->GetModuleByPath(function.second.loaded_module_path());
+    instrumented_function.set_file_path(function.second.module_path());
+    const ModuleData* module_data = module_manager->GetModuleByPathAndBuildId(
+        function.second.module_path(), function.second.module_build_id());
+
+    // In the case module_data was not found by build id check if FunctionInfo build_id is empty,
+    // in which case assume this is capture saved with Orbit v1.61 and below try to get module
+    // build id from path and do another lookup.
+    if (module_data == nullptr && function.second.module_build_id().empty()) {
```

Comment:
We should probably add a test for both cases?

Reply:
done.